pom.xml

```diff
@@ -1791,7 +1791,6 @@
             <log4j.configurationFile>${test.log4j.scheme}${test.conf.dir}/hive-log4j2.properties</log4j.configurationFile>
             <hive.test.console.log.level>${test.console.log.level}</hive.test.console.log.level>
             <hive.cluster.id>hive-test-cluster-id-cli</hive.cluster.id>
-            <log4j.debug>true</log4j.debug>
```

Comment:
Why the change?

Reply:
log4j.debug actually has no effect in previous versions of log4j2. Setting it to `true` will also make it impossible to override changes to log4j2's internal behavior through the command line. In log4j 2.24.0, which strengthens the role of log4j.debug, setting log4j.debug will cause the configuration of log4j2's internal log in the log4j2 properties file to become invalid. log4j.debug should usually only be called from the command line.

Reply:
Did you see any issue with `log4j.debug` in your change?

how we use `log4j.debug` from command?  -Dlog4j.debug=true ?

Reply:
> Did you see any issue with `log4j.debug` in your change?

- The full error stack is at https://github.com/apache/logging-log4j2/issues/2794 .

> how we use `log4j.debug` from command?  -Dlog4j.debug=true ?

- Yes.